src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -147,6 +147,7 @@ var HTMLDOMPropertyConfig = {
     srcDoc: 0,
     srcLang: 0,
     srcSet: 0,
+    srcObject: 0,
```

Comment:
It looks like `srcObject` should be set as a property on the element, not as an attribute. We should use `MUST_USE_PROPERTY` instead of `0` here to force React to set it as a property on the element.